operator/src/lib.rs

```diff
@@ -453,11 +451,14 @@ fn build_pod(
             containers: vec![Container {
                 image: Some(format!(
                     "stackable/kafka:{}",
-                    serde_json::json!(resource.spec.version).as_str().unwrap()
+                    &resource.spec.version.get_fully_qualified_version()
                 )),
                 name: "kafka".to_string(),
                 command: Some(vec![
-                    "kafka_2.12-2.6.0/bin/kafka-server-start.sh".to_string(), // TODO: Don't hardcode this
+                    format!(
+                        "kafka_{}/bin/kafka-server-start.sh",
+                        &resource.spec.version.get_fully_qualified_version()
```

Comment:
No need to dereference the `String`.
```suggestion
                        resource.spec.version.get_fully_qualified_version()
```

Reply:
fixed